src/FullBuild/Core/Simplify.fs

```diff
@@ -160,11 +182,12 @@ let SimplifyAnthologyWithoutPackage (antho : Anthology) =
                      with Projects = newProjects }
     newAntho
 
-let SimplifyAnthologyWithPackages (antho : Anthology) (package2files : Map<PackageId, AssemblyId set>) (package2packages : Map<PackageId, PackageId set>) =
+let SimplifyAnthologyWithPackages (antho : Anthology) (package2files : Map<PackageId, AssemblyId set>) (package2packages : Map<PackageId, PackageWithDependencies>) =
+    let package2packagesIgnoreFramework = package2packages |> Map.map (fun projectId dependencies -> dependencies.Dependencies |> Seq.collect(fun d-> d.Dependencies) |> Set.ofSeq)
     let newProjects = antho.Projects |> TransformSingleAssemblyToProjectOrPackage  package2files
                                      |> TransformPackageToProject
-                                     |> TransformPackagesToProjectsAndPackages package2packages package2files
-                                     |> RemoveAssembliesFromPackagesOrProjects package2files
+                                     |> TransformPackagesToProjectsAndPackages package2packagesIgnoreFramework package2files
```

Comment:
package2packagesIgnoreFramework is wrong so packages are not correctly removed then.